app/assets/stylesheets/action_page.css.scss

```diff
@@ -377,8 +374,10 @@
 .description h3, h3.privacy-notice-header,
 h3.header, #tools #email-tool h3 {
   position: relative;
-  margin-bottom: 1em;
+  margin-bottom: 1em !important;
```

Comment:
Instead of using !important here, perhaps you could make a change to line 589 action_page.css.scss to increase (or not define) the margin there.